packages/base-controller/jest.config.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * For a detailed explanation regarding each configuration property and type check, visit:
+ * https://jestjs.io/docs/configuration
+ */
+
+import merge from 'ts-deepmerge';
+import baseConfig from '../../jest.config.packages';
+
+const config = merge(baseConfig, {
+  // The display name when running multiple projects
+  displayName: 'base-controller',
```

Comment:
Since we are using Jest projects, and each project gets run in its own runner, it's helpful to know which set of files belong to which project. This `displayName` will show up as a little box next to the file being run when running `yarn jest`.